validator/sawtooth_validator/journal/journal.py

```diff
@@ -125,19 +128,23 @@ def __init__(self,
         self._chain_thread = None
 
     def _init_subprocesses(self):
+        state_view_factory = StateViewFactory(self._merkle_db)
```

Comment:
Why not just pass in the `state_view_factory`?  Journal doesn't need access to the merkle_db.